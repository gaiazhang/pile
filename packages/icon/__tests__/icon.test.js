/**
 * @author renmaomin@126.com
 */
import React from 'react';
import { shallow } from 'enzyme';
import Icon from '../src';

describe('Icon suite', () => {
  it('render a Icon', () => {
    const wrapper = shallow(<Icon type="delete" />);
    expect(wrapper).toMatchSnapshot();
  });
});
