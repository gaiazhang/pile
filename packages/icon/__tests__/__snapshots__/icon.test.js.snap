// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon suite render a Icon 1`] = `
<i
  className="pile-icon-delete"
/>
`;
